docs/decisions/0014-enterprise-catalog-sync-catalogs.rst

```diff
@@ -0,0 +1,43 @@
+Catalog syncing behavior with enterprise-catalog service
+------------------------------------------------------------------------------
+
+Status
+======
+
+Accepted
+
+Context
+=======
+ECS team would like to have a way to reuse a saved set of courses configured by a query and use them in multiple catalogs across customers.
+
+
+Decisions
+=========
+
+- **Sync catalog/customer data with enterprise-catalog**
+As the catalog data is saved in both edx-enterprise and enterprise-catalog, we need to make sure that catalog data is consistent across both.
+In the context of edx-enterprise, changes to EnterpriseCatalogQuery and EnterpriseCustomerCatalog will be propagated to their counterparts
+in the enterprise-catalog service.
+
+
+Consequences
+============
+
+- **EnterpriseCatalogQuery Sync Process**
+The enterprise-catalog service enforces a uniqueness constraint for the `content_filter` field, and so edx-enterprise must enforce it as well
+or the sync operation will fail and the entities will be out of sync with each other.
+
+In the admin console, whenever a new EnterpriseCatalogQuery is created, or an existing one edited, we first query enterprise-catalog to see if its 
```

Comment:
nit: I would say specifically "Django Admin" just to help disambiguate from the admin portal.